### ISSUE TITLE
Title and role attributes are missing for functional control  in Design

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#783]
**Describe the bug**
Title and role attributes are missing for functional control  in Design screen. In Design View, there are arrow icons on the left and right side of the carousel. When we move mouse over it, there is not label/title displayed. The same exists for "open page" and "change template" option for pages, on same screen etc.

**To Reproduce**
1. Go to Design (http://localhost:9992/cm/app/?view=design&site=test&path=%2FSites%2Ftest%2F)
2. Double Click on any site.
3. Mouse over on  the right and left arrow icons placed in page carousel.
4. No title/label displayed.
5. Same is true for other anchor tags/divs and other visible elements on page.

**Expected behavior**
Title/Label should be displayed on each component of the screen.

**Comment by ramdev1980** [comment]
Making changes for div components inside Explore region under layout tab, in design view, but not reflecting.

**Comment by ramdev1980** [comment]
Making changes for div components inside Explore region under layout tab, in design view, but not reflecting.

**Comment by ramdev1980** [comment]
completed
